index.js

```diff
@@ -433,6 +433,11 @@ module.exports = (params) => {
     AWS.config.update({ httpOptions: { agent: sslAgent } })
   }
 
+  // Update the AWS http agent with the region
+  if (typeof params.region !== false) {
+    AWS.congif.update({ region: params.region });
```

Comment:
I presume it should be config :)

Reply:
Not sure how that made it through - I'll push the change